guessit/__main__.py

```diff
@@ -213,8 +213,6 @@ def main(args=None, setup_logging=True):
         # Wrap sys.stdout into a StreamWriter to allow writing unicode.
         sys.stdout = codecs.getwriter(locale.getpreferredencoding())(sys.stdout)
 
-    from guessit.plugins import transformers
```

Comment:
I don't think you can remove this line: although it looks like this doesn't do anything, it actually does! When importing this module, the file ``guessit/plugins/transformers.py`` gets executed, and at the end of this file here: https://github.com/wackou/guessit/blob/master/guessit/plugins/transformers.py#L219 the ``reload()`` function is called. It wouldn't be called if the file is not imported...